### ISSUE TITLE
Unexpected padding between formulas

### ISSUE THREAD

**Issue by ringosham** [noties/Markwon#138]
* **Markwon version**: 3.0.2
* **Markwon Latex plugin version**: 3.0.2
* Compiled SDK version: 28
* Android version: Pie (9)

1. Please specify expected/actual behavior
There's extra padding between the formulas where there should be none.
![Screenshot_20190619-160606](https://user-images.githubusercontent.com/7325740/59777282-49b2de00-92ac-11e9-8b64-fbff96f0b699.jpg)

2. Please specify conditions/steps to reproduce (layout, code, markdown used, etc)
This is the code used in the activity.
```
@Override
protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
    TextView text = findViewById(R.id.test);
    Markwon markdown = Markwon.builder(this).usePlugin(ImagesPlugin.create(this)).usePlugin(JLatexMathPlugin.create(50)).build();
    markdown.setMarkdown(text, readMarkdownFile(R.raw.markdown));
}

public String readMarkdownFile(int resourceId) {
    InputStream in = getResources().openRawResource(resourceId);
    Scanner scan = new Scanner(in).useDelimiter("\\A");
    return scan.hasNext() ? scan.next() : "";
}
```

The markdown used.
```
$$
  P(X=r)=\frac{\lambda^r e^{-\lambda}}{r!}
$$

$$
  P(X<r)=P(X<r-1)
$$

$$
  P(X>r)=1-P(X<r=1)
$$

$$
  \text{Variance} = \lambda
$$
```

**Comment by noties** [comment]
Hello @ringosham !

I can reproduce this issue. This happens because `JLatexMathPlugin` relies on `ImagesPlugin` (all latex is displayed via `AsyncDrawable`). And internally we set `image-size` for a latex formula to be 100% of canvas width. Thus, image is scaled up whilst keeping the ratio (increase in both width and height), but this scaling is ignored by `JLatexMathDrawable`.

The solution would be of cause to stop relying on ImagesPlugin. And the good thing is: it is already done in upcoming `4.0.0` snapshot version. I'm going to fix this issue also. So, if you are only evaluating the project starting with the next `4.0.0` version might be a good idea (less migration).

I have also noticed that `JLatexMathPlugin` uses the same drawable for all latex formulas background and going to fix that also.

[v4.0.0](https://noties.io/Markwon/docs/v4/install.html)
